3x-ui/umbrel-app.yml

```diff
@@ -0,0 +1,53 @@
+manifestVersion: 1
+id: 3x-ui
+name: 3x-UI
+tagline: Xray panel supporting multi-protocol multi-user expire day & traffic & ip limit (Vmess & Vless & Trojan & ShadowSocks & Wireguard)
```

Comment:
Please try to keep the tagline under 100 characters as it would overflow otherwise in the apps overview